src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ReadyToRunHelperNode.cs

```diff
@@ -96,6 +95,30 @@ public override string MangledName
                         return "__InterfaceDispatch_" + NodeFactory.NameMangler.GetMangledMethodName((MethodDesc)_target);
                     case ReadyToRunHelperId.ResolveVirtualFunction:
                         return "__ResolveVirtualFunction_" + NodeFactory.NameMangler.GetMangledMethodName((MethodDesc)_target);
+                    case ReadyToRunHelperId.GenericLookupFromThis:
+                        {
+                            var lookupInfo = (GenericLookupDescriptor)_target;
```

Comment:
@sergiy-k It does, but I always open a new scope when the switch statement declares locals because of C#'s unfortunate choice of not doing it by default:

```csharp
        switch (X)
        {
            case 0:
                int x = 1234;
                // ...
            case 1:
                int x = 5678;
                // ...
        }
```

Won't compile.

Not opening a new scope leads to choices of weird local variable names in the long term (purely because the apt one is already in use by a different case block). On the other hand putting one-liners in a new scope is a waste of space. The only winning strategy is not to play.